group_vars/pdc_describe/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-postgres_version: 13
+postgres_version: 15
```

Comment:
Won't this change production as well as staging? I don't think we're ready to change both.

Reply:
this only updates the client. This https://github.com/pulibrary/princeton_ansible/issues/4105 will document how this role functions.